pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -280,3 +279,18 @@ func (d *dashboardUidPermissionMigrator) migrateIdScopes(sess *xorm.Session) err
 	}
 	return nil
 }
+
+type managedFolderCreateAction struct {
+	migrator.MigrationBase
+}
+
+func (m *managedFolderCreateAction) SQL(dialect migrator.Dialect) string {
+	return CodeMigrationSQL
+}
+
+func (m *managedFolderCreateAction) Exec(sess *xorm.Session, migrator *migrator.Migrator) error {
+	if _, err := sess.Exec("DELETE FROM permission WHERE action = 'folders:create' AND scope LIKE 'folders:uid:%'"); err != nil {
```

Comment:
question: I guess we're assuming that for custom roles with `folders:create` action the scope won't be applied? Just want to make sure that we're not potentially messing up custom roles with this migration.